### PR TITLE
First attempt to resolve issue 995

### PR DESCRIPTION
**Pull request by ndw**
This is my first attempt to clarify the binding of the default readable port per #995. At the moment, I've only added some prose to the "Connections" section.

### CODE REVIEW

**Review by gimsieke — COMMENTED** [review]
This is fine, but I think we might add another example or note. A scenario as @dmj described it in https://twitter.com/_dmaus/status/1301537247079079938. If you have a pipeline that copies file A to B and then B to C, it might throw an error because it attempts to execute the B→C operation before the A→B operation has finished or even started. `p:file-copy` has a primary output but the next copy operation’s `@href` and `@target` attributes are just string constants, they don’t depend on the previous step’s output. Without explicit dependency declaration, the steps might be executed in parallel or in “wrong” order.

**Comment by ndw** [comment]
Quite right. I added those examples and made some clarifying editorial changes.

File: xproc/src/main/xml/specification.xml [hunk 1]
```diff
@@ -1179,12 +1179,13 @@ sibling. Consider the following fragment:</para>
                  attribute-value="{count(/*/*)}"/>]]></programlisting>
 
 <para>The output from the add XML base step is the natural input to the
-add-attribute step. That’s the source for both the document that will
-be processed by the step and the document that will be used as the
+add-attribute step. The add XML step is the source for both the document that will
```
Comment (gimsieke) [inline xproc/src/main/xml/specification.xml:1182]: add XML step → add XML base step

**Review by gimsieke — APPROVED** [review]

File: xproc/src/main/xml/specification.xml (hunk 1 above)
Comment (ndw) [reply xproc/src/main/xml/specification.xml:1182]: 👍

**Pull request merged**